REBRANDED_OUTPUT/scripts/subagents/check-content-integrity.sh

```
@@ -0,0 +1,345 @@
+    done
+}
+
+# Check image references
+check_image_references() {
+    log_section "Checking Image References"
+
+    log_info "Scanning XHTML files for image references..."
+
+    local broken_refs=0
+
+    for xhtml_file in xhtml/*.xhtml templates/*.xhtml 2>/dev/null; do
```

Shellcheck (0.11.0)

[error] Couldn't parse this for loop. Fix to allow more checks.

(SC1073)